velox/exec/tests/TaskTest.cpp

```diff
@@ -740,6 +741,22 @@ DEBUG_ONLY_TEST_F(TaskTest, liveStats) {
     EXPECT_EQ(3 * i, operatorStats.outputPositions);
     EXPECT_EQ(i, operatorStats.outputVectors);
     EXPECT_EQ(0, operatorStats.finishTiming.count);
+
+    EXPECT_EQ(1, liveStats[i].totalDrivers);
+    EXPECT_EQ(0, liveStats[i].completedDrivers);
+    EXPECT_EQ(0, liveStats[i].terminatedDrivers);
+    EXPECT_EQ(1, liveStats[i].runningDrivers);
+    for (auto& blocked : liveStats[i].blockedDrivers) {
+      EXPECT_EQ(0, blocked);
+    }
+  }
+
+  EXPECT_EQ(1, finishStats.totalDrivers);
+  EXPECT_EQ(1, finishStats.completedDrivers);
+  EXPECT_EQ(0, finishStats.terminatedDrivers);
+  EXPECT_EQ(0, finishStats.runningDrivers);
+  for (auto& blocked : finishStats.blockedDrivers) {
+    EXPECT_EQ(0, blocked);
```

Comment:
Could you add a test case that having a driver to be blocked for something. Like take hash join and blocked the hash build and check that all hash probes are blocked waiting for kWaitForJoinBuild? Thanks!

Reply:
I would like some pointers here, either to an existing test or how to create one that will block. Thanks!